Added cosign signatures support. (#1199)
pulp/pulp_container: docs/workflows/helm-support.rst

    $ helm registry login pulp3-source-fedora36.puffy.example.com

Push the chart to your Pulp Container registry using the helm push command::

    $ helm push etherpad-0.0.4.tgz oci://pulp3-source-fedora36.puffy.example.com
    Pushed: pulp3-source-fedora36.puffy.example.com/etherpad:0.0.4
    Digest: sha256:a6667ff2a0e2bd7aa4813db9ac854b5124ff1c458d170b70c2d2375325f2451b

Ensure that the push worked by deleting the local copy, and then pulling the chart from the repository::

    $ rm -rf etherpad-0.0.4.tgz

    $ helm pull oci://pulp3-source-fedora36.puffy.example.com/etherpad --version 0.0.4
    Pulled: pulp3-source-fedora36.puffy.example.com/etherpad:0.0.4
    Digest: sha256:4f627399685880daf30cf77b6026dc129034d68c7676c7e07020b70cf7130902

The chart can then be installed using the helm install command::

    $ helm install etherpad-0.0.4.tgz

Alternatively, charts can be installed directly from the registry without needing to download locally.
Use the helm install command and reference the registry location::

    $ helm install oci://pulp3-source-fedora36.puffy.example.com/helm/etherpad --version=0.0.4

Mirror
------

> Being an OCI compliant registry Pulp Container registry can natively mirror helm charts
> wich are stored as an OCI image::

Reviewer: ```suggestion
As an OCI-compliant registry, the Pulp Registry can natively mirror helm charts
which are stored as an OCI image::
```
